frontend/src/components/Menu.vue

```diff
@@ -6,23 +6,70 @@
 -->
 
 <template>
-  <div @click.stop="hide" class="dimmed no-scroll">
-    <div class="vertical-container">
-      <RouterLink to="/about" class="link">
-        <label>About</label>
-      </RouterLink>
-      <!-- <RouterLink to="/" class="link">
-        <label>Events</label>
-      </RouterLink> -->
-      <RouterLink to="/resources" class="link">
-        <label>CSE Resources</label>
-      </RouterLink>
-      <RouterLink to="/sponsors" class="link">
-        <label>Sponsors</label>
-      </RouterLink>
-      <RouterLink to="/engage" class="link">
-        <label>Engage</label>
+  <div @click.stop="hide" class="cover">
+    <div class="frame">
+      <!-- Logo -->
+      <RouterLink to="/" v-on:click.native="showMenu = false">
+        <div class="fill-height" style="width: 140px; margin-left: 5px;">
+          <v-img src="@/assets/csesoc-logo-white.svg" />
+        </div>
```

Comment:
This is not displayed.

Reply:
This logo should be visible 🤔 it's imported the same way as the main logo on our navbar. Maybe take make sure that the SVG is actually in the assets folder.